nixos/modules/module-list.nix

```diff
@@ -260,6 +260,8 @@
   ./programs/oblogout.nix
   ./programs/oddjobd.nix
   ./programs/openvpn3.nix
+  ./programs/obs-studio.nix
+  ./programs/pantheon-tweaks.nix
```

Comment:
What is that `pantheon-tweaks` ?

Reply:
it was a merge-conflict I fixed using the github website, let me rebase it